translations/ja.json

```
@@ -954,64 +954,64 @@
+	"usage: minikube config unset PROPERTY_NAME": "使用法: minikube config unset PROPERTY_NAME",
+	"usage: minikube delete": "使用法: minikube delete",
+	"usage: minikube profile [MINIKUBE_PROFILE_NAME]": "使用法: minikube profile [MINIKUBE_PROFILE_NAME]",
+	"using metrics-server addon, heapster is deprecated": "metrics-server アドオンを使用します (heapster は廃止予定です)",
+	"version json failure": "JSON 形式のバージョン表示に失敗しました",
+	"version yaml failure": "YAML 形式のバージョン表示に失敗しました",
+	"zsh completion failed": "zsh のコマンド補完に失敗しました",
+	"zsh completion.": "zsh のコマンド補完です。",
+	"{{ .name }}: Suggestion: {{ .suggestion}}": "{{ .name }}: 提案: {{ .suggestion}}",
+	"{{ .name }}: {{ .rejection }}": "",
+	"{{.Driver}} is currently using the {{.StorageDriver}} storage driver, consider switching to overlay2 for better performance": 	"{{.Driver}} は現在 {{.StorageDriver}} ストレージドライバーを使用しています。性能向上のため overlay2 への切替を検討してください",
+	"{{.cluster}} IP has been updated to point at {{.ip}}": "{{.cluster}} の IP アドレスは {{.ip}} に更新されました",
+	"{{.cluster}} IP was already correctly configured for {{.ip}}": "{{.cluster}} の IP アドレスはすでに {{.ip}} に設定されています",
+	"{{.count}} nodes stopped.": "{{.count}} 台のノードが停止しました。",
+	"{{.count}} node{{if gt .count 1}}s{{end}} stopped.": "{{.count}} 台のノードが停止しました。",
 	"{{.driver_name}} \"{{.cluster}}\" {{.machine_type}} is missing, will recreate.": "{{.driver_name}} 「 {{.cluster}} 」 {{.machine_type}} がありません。再生成します。",
-	"{{.driver_name}} couldn't proceed because {{.driver_name}} service is not healthy.": "",
-	"{{.driver_name}} has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "",
-	"{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "",
+	"{{.driver_name}} couldn't proceed because {{.driver_name}} service is not healthy.": "{{.driver_name}} サービスが正常ではないため、{{.driver_name}} は機能しません。",
+	"{{.driver_name}} has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "{{.driver_name}} で利用できる CPU が 2 個未満ですが、Kubernetes を使用するには 2 個以上の CPU が必要です",
+	"{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "{{.driver_name}} は {{.container_limit}}MB のメモリーしか使用できませんが、{{.specified_memory}}MB のメモリー使用を指定されました",
 	"{{.driver}} does not appear to be installed": "{{.driver}} がインストールされていないようです",
-	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロフィールから指定されています。「 minikube delete 」を実行、あるいは {{.driver}} をインストールしてください",
-	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
+	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロフィールから指定されています。'minikube delete' を実行するか、{{.driver}} をインストールしてください",
```

```suggestion
	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロファイルから指定されています。'minikube delete' を実行するか、{{.driver}} をインストールしてください",
```
他の和訳と合わせるため、「プロフィール」→「プロファイル」がよいです。